internal/grpctest/grpctest.go

```diff
@@ -48,16 +49,22 @@ type Tester struct{}
 // Setup updates the tlogger.
 func (Tester) Setup(t *testing.T) {
 	TLogger.Update(t)
+	// TODO: There is one final leak around closing connections without completely
```

Comment:
Is this TODO already addresses? I don't see any commented out code here.

Reply:
So we talked about this in the meeting, I will add a per-test leakcheck override instead of globally disabling it.